src/cbor_service_helper.c

```diff
@@ -0,0 +1,160 @@
+#include "cbor_service_helper.h"
+#include "bm_os.h"
+#include "crc.h"
+#include "messages/config.h"
+#include <inttypes.h>
+#include <stdlib.h>
+#include <string.h>
+
+uint8_t *services_cbor_as_map(size_t *buffer_size, BmConfigPartition type) {
```

Comment:
This function was refactored to utilize the already created API @victorsowa12 created in `bm_configs_generic.h`.